### ISSUE TITLE
video game sales

### ISSUE THREAD

**Issue by Sree-Ranjini** [MicrosoftDocs/powerbi-docs#4627]
[Enter feedback here]

please provide sample pbix for all your pages. eg. In this page you refer to video game sales but the corresponding pbix is missing. Its hard to follow  and not possible to practice
---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: f919f51d-8623-b039-a1fc-0df774225c46
* Version Independent ID: 0376c331-0f7e-ec9b-6262-f47a6c413d47
* Content: [Decomposition tree - Power BI](https://learn.microsoft.com/en-us/power-bi/visuals/power-bi-visualization-decomposition-tree)
* Content Source: [powerbi-docs/visuals/power-bi-visualization-decomposition-tree.md](https://github.com/MicrosoftDocs/powerbi-docs/blob/main/powerbi-docs/visuals/power-bi-visualization-decomposition-tree.md)
* Service: **powerbi**
* Sub-service: **powerbi-ai**
* GitHub Login: @mihart
* Microsoft Alias: **mihart**

**Comment by bishalgoswami** [comment]
Thanks for your comment, @TechnicalCode. This issue has been assigned to the doc author for review.

**Comment by davidiseminger** [comment]
Thanks for your feedback. Unfortunately we do not have .pbix files for all of the content we create, but some sample files are available.
